storage/coop/rchain/key-value-store/Key.scala

```diff
@@ -7,20 +7,20 @@
 
 package KeyValueStore
 
-import scala.collection.mutable._
+import scala.collection.mutable.{ArrayBuffer, LinkedHashSet}
```

Comment:
Our [codified dogma](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards) states that
> When using collections, qualify names by importing scala.collection.immutable and/or scala.collection.mutable
e.g. "immutable.Map"

Reply:
I poured the code files into the structure above.  If all my scala files are in src/main/scala/coop/rchain/key-value-store then the tests run in sbt.  If I move Tests.scala to src/test/scala/coop/rchain/key-value-store, then the code won't compile with the error that the objects Tests and TestTools can't be found.

I'm searching for a solution, including manipulating the "mainClass in (Compile,run)" parameter in build.sbt, but havent' solved it yet.  Do you know what may be required to make sbt find the code in src/test/scala/coop/rchain/key-value-store?

Reply:
The easiest solution is to rename the `key-value-store` directory to `KeyValueStore` so it matches the package name, I think.

Reply:
Yes, I tried that but didn't mention it the message above.  I'm going to try the fully qualified package src.main.scala.coop.rchain.KeyValueStore now.